Humidifier: add purpose-specific triggers (Labs feature) (#45071)
home-assistant/home-assistant.github.io: source/_triggers/humidifier.turned_off.markdown

YAML sometimes provides additional options for more complex use cases that are not available through the UI.

{% options_yaml %}
behavior:
  description: |
    When multiple humidifiers are targeted, controls when the trigger fires:

    - `any` (**Each** in the UI, default): fire every time any targeted humidifier turns off.
    - `first` (**First** in the UI): fire only when the first of a group turns off.
    - `last` (**All** in the UI): fire only after every targeted humidifier is off.
  required: true
  type: string
  default: any
for:
  description: |
    How long the humidifier must stay off before the trigger fires. Accepts a duration string in `HH:MM:SS` format. For example, `00:05:00` fires only after the humidifier has stayed off for 5 minutes.
  required: true
  type: string
  default: "00:00:00"
{% endoptions_yaml %}

{% include triggers/targets.md %}

{% include triggers/behavior.md %}

## Good to know

- The trigger only fires when a humidifier transitions from a known, valid state. Transitions from the **Unavailable** or **Unknown** state to off do not count.
- To react to the opposite transition, use [Humidifier turned on](/triggers/humidifier.turned_on/).

> - Pair **Humidifier turned off** with the `last` behavior to run something only once every humidifier in an area has stopped, like sending a single notification when all humidifiers in the house are off.

Reviewer: Let's break down this sentence in two:

```suggestion
- Pair **Humidifier turned off** with the `last` behavior to do something only after every humidifier in an area has stopped. For example, send a single notification when all humidifiers in the house are off.
```
